pulseaudio-control.bash

```diff
@@ -320,6 +348,30 @@ function usage() {
     echo "    https://github.com/marioortizmanero/polybar-pulseaudio-control"
 }
 
+while [[ "$1" = --* ]]; do
+    case "$1" in
+        --vol-icon-low=*)
+            VOLUME_ICONS[0]="${1#--vol-icon-low=}"
+            ;;
+        --vol-icon-mid=*)
+            VOLUME_ICONS[1]="${1#--vol-icon-mid=}"
+            ;;
+        --vol-icon-high=*)
+            VOLUME_ICONS[2]="${1#--vol-icon-high=}"
+            ;;
+        --sink-icon=*)
```

Comment:
Is it me or does this not work with arguments like `--arg value`? It only seems to work for `--arg=value`. If anything, I'd just enforce the first so that this is easier (you just match `--sink-icon` and read `$2` for the value).

Reply:
No, it doesn't as is. I prefer `=` forms personally, and in this rudimentary parsing it makes it more robust against mis-invocations like `--arg1 --arg2 val2` (missing value to arg1), but will change required `=` to required ` ` if you prefer.

I've refactored to support both (and it's a bit tidier anyway :slightly_smiling_face:).

Reply:
Your refactor looks great but it won't work when we have flags instead of arguments. For example, `--notifications` can be passed without a value, and your script will shift twice and skip the next argument.

Sorry if I'm making this too hard it's just that bash kinda sucks once the script becomes larger. At some point Python becomes a better choice haha

Reply:
:+1: good spot, sorted